### PR TITLE
Build gumbo with VC++2015(v140) and build x64 library

### PR DESCRIPTION
**Pull request by ZigZagT**

### CODE REVIEW

**Comment by 4Oranges** [comment]
I signed it!

File: src/gumbo.h [hunk 1]
```diff
@@ -36,7 +36,7 @@
  * @endcode
  * HTML5 Spec:
  *
- * http://www.whatwg.org/specs/web-apps/current-work/multipage/syntax.html
+ * http://www.whatwg.org/specs/web-apps/ui-work/multipage/syntax.html
```
Comment (nostrademons) [inline src/gumbo.h]: Curious about the comment changes in here and other source files.  Have the spec files been moved?  For me, the new links 404, while the old ones still work.  Or is it a simple find & replace snafu.

File: src/tokenizer.c [hunk 2]
```diff
@@ -82,14 +82,14 @@ typedef struct GumboInternalTagState {
   // of the buffer.
   const char* _original_text;
 
-  // The current tag enum, computed once the tag name state has finished so that
+  // The ui tag enum, computed once the tag name state has finished so that
```
Comment (nostrademons) [inline src/tokenizer.c]: Ditto here.

Comment (4Oranges) [inline src/tokenizer.c]: Yes it's a simple find&replace. Im using gumbo as a submodule in my own project, and modified these files by accident when i was doing a refactoring of my code. And a later commit had restored those links to there original value.

**Pull request closed without merging**